docs/pages/use-teleport/teleport-connect.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Using Teleport Connect
+description: Using Teleport Connect
+---
+
+Teleport Connect provides easy and secure access to SSH servers and databases, with support for
+other resources such as Kubernetes clusters and application access coming in the future.
+
+![resources tab in Teleport Connect](../../img/use-teleport/connect-cluster.png)
+
+## Installation & Upgrade
+
+Head over to the [Downloads](https://goteleport.com/download/) page to download the most recent
+version of Teleport Connect. As of now, Teleport Connect supports macOS only. Double-click the
+downloaded .dmg file and drag the Teleport Connect icon to the Applications folder.
+
+To upgrade Teleport Connect to a newer version, drag the new version to the Applications folder.
+
+## Interface Overview
+
+![interface overview of Teleport Connect](../../img/use-teleport/connect-ui-overview.png)
+
+The top bar of Teleport Connect consists of:
+
+- The profile selector (the top right) allowing you to switch between profiles on different Teleport
+  clusters as well as log in or out of the clusters.
+- The connection list (the top left) showing recent connections, allowing you to seamlessly switch
+  between them.
+- The command bar (in the middle) which launches any command you input in a new terminal tab.
+- The cluster selector (to the left of the command bar) shows up only if there are leaf clusters
+  connected to the root cluster. It lets you browse leaf cluster resources. It also changes
+  which cluster the commands in the command bar are going to target.
+
+## Connecting to an SSH Server
```

Comment:
It would be great to have more UI screenshots in the sections below this point:
- Choosing an SSH server to connect to
- Choosing a database to connect to
- A relevant screenshot for the "GUI client" section
- A relevant screenshot for the "CLI Client" section
- Choosing a cluster to connect to (in "Connecting to multiple clusters")

If adding these screenshots won't be possible in the time we have, we can open a GitHub issue to do this later.

Reply:
> If adding these screenshots won't be possible in the time we have, we can open a GitHub issue to do this later.

Yeah, unfortunately I definitely won't have time to add those screenshots right now.

In general, we wanted keep the screenshots to minimum as we plan to substantially change the UI in the future. But that probably won't happen until another month or two if not more.

Reply:
Makes sense! We can hold off on the screenshots for this release